codecov.yml

```diff
@@ -0,0 +1,15 @@
+codecov:
```

Comment:
most of the file have `.yaml` extension not `.yml` would be nice to be consistent